makefiles/Makefile.devhub-content

```diff
@@ -62,13 +62,14 @@ next-gen-stage: ## Host online for review
 	@:
 
 next-gen-deploy:	
-	@yes | mut-publish public ${PRODUCTION_BUCKET} --prefix="$(filter-out $@,$(MAKECMDGOALS))" --deploy --deployed-url-prefix=https://docs.mongodb.com --json --all-subdirectories ${ARGS};
-	@echo "Hosted at ${PRODUCTION_URL}/$(filter-out $@,$(MAKECMDGOALS))/index.html";
-	$(MAKE) deploy-search-index
```

Comment:
removed the call to deploy-search-index bc I checked previous version of this makefile and saw the deploy target did not call deploy-search-index.